transfermethodui/src/androidTest/java/com/hyperwallet/android/ui/transfermethod/SelectTransferMethodTest.java

```diff
@@ -106,6 +127,9 @@ public void testSelectTransferMethod_verifyCorrectLabelsDisplayed() {
 
         mActivityTestRule.launchActivity(null);
 
+        verify(mHyperwalletInsight, atLeastOnce()).trackImpression(any(Context.class), anyString(), anyString(),
```

Comment:
For all of these automated test cases we need to check that the function is called with the specific parameters related to the test.  For example in this one:
- pageName: transfer-method:add:select-transfer-method
- pageGroup: transfer-method
- Country: country of list shown
- Currency: currency of list shown

I'd suggest capturing the map to check that the country and currency are in the parameters

Reply:
Updated parameters.